blog.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adopt A Dog - Blog</title>
+  </head>
+
+  <body style="background-color:powderblue;">
+
+<!-- This is the header on the page - in two sections for styling later.  -->
+    <header>
+      <article id="left-header">
+        <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Logo for Adopt A Dog with silouette of dog."></a>
+        <img src="images/header-image.jpg" alt="Background image of brown dog.">
+        <h2 class="header-text">Seattle's premiere dog adoption agency</h2>
+      </article>
+
+      <article id="right-header">
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="Icon of a shopping cart.">$1234.56</a>
+        <p class="checkout"><a href="checkout.html">Checkout</a></p>
+      </article>
+    </header>
+
+<!--This is the navigation - current page has id of active for styling in navigation later-->
+    <ul>
+      <li class="navigation"><a href="index.html">Home /</a></li>
+      <li class="navigation"><a href="dogs.html">Dogs /</a></li>
+      <li class="navigation"><a id="active" href="blog.html">Blog</a></li>
+    </ul>
+
+<!-- Section for blog posts -->
+    <section id="blog-content" class="grid-container-blog">
```

Comment:
Great use of section. The id is not really needed. Best practice is to reserve id's for javascript purposes and classes for styling (well...and JS too...) It's totally acceptable to declare a class and use it once in lieu of a id

Reply:
Got it.